components/eam/src/physics/crm/rrtmg/radiation.F90

```diff
@@ -2106,21 +2100,6 @@ subroutine radiation_tend( state, ptend,pbuf, &
                          clm_seed,     lu,           ld                                            )
                 call t_stopf ('rad_rrtmg_lw')
 
-                if (lwrad_off) then
```

Comment:
just make sure variables enclosed in if(lwrad_off) block is initialized to zero elsewhere.

Reply:
I'm pretty sure we don't actually need to worry about these when we are not running radiation. The only one I'm not sure about is the cam_out variable, but I'm not sure there's any situation where we would be running an active surface component without any radiation calculations. I'm gonna omit zeroing these out for now and we can come back and fix it if it breaks one of the SCM cases.

Reply:
If these vars are not needed, it seems , then would it be better to remove them from declarations? Same for cam_out%flwds . if you remove their init, I assume running with debug (which probably has -init=snan ) would help to see if these are used.

Reply:
The variables are definitely used in typical use cases for diagnostic output, but the only cases that require disabling radiation are the handful of SCM cases that have the radiative tendencies included in the forcing. Ben and I had a long discussion about this and we agreed that if the variables are not already properly initialized then that is outside the scope of this PR. 

Additionally, we're not sure whether the step to zero out all the variables is actually necessary or not, and I'm not exactly sure how to test it because I'm not sure any of the SCM cases require the combination of an active surface component and disabled radiation, which seems to be the only way it would be problematic. 

So I feel that it's better to remove this now and then implement a proper initialization block later if a problem surfaces.